Jenkinsfile

```diff
@@ -10,7 +10,16 @@ pipeline {
     disableConcurrentBuilds(abortPrevious: true)
   }
   environment {
+    AR_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-09-23-1'
+    DE_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-09-23-1'
     EN_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-03-23-1'
+    ES_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-09-23-1'
+    FR_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-09-23-1'
+    PT_TN_CACHE='/home/jenkinsci/TestData/text_norm/ci/grammars/02-09-23-1'
```

Comment:
could you also add swedish, russion, chinese, vietnamese?

Reply:
sure, once the initial set of tests run

Reply:
Done